.github/workflows/on-push.yml

```diff
@@ -0,0 +1,23 @@
+name: on-push
+on: [push]
+jobs: 
+  lerna-build: 
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with: 
+          node-version: '14'
+      - run: npx lerna bootstrap
+      - run: npx lerna run build 
+  lerna-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with: 
+          node-version: '14'
+      - run: npm install typescript --save-dev
+      - run: npm install @typescript-eslint/eslint-plugin@latest --save-dev
+      - run: npm install @typescript-eslint/parser@latest --save-dev
```

Comment:
```suggestion
      - run: npm install 
```

Reply:
done in [updated node version and commands in actions] commit